PROJECT.md

```diff
@@ -2,34 +2,34 @@
 
 Each project repository **requires**, at a minimum:
 
-1. Readme -- `README.md`
-1. License -- `LICENSE`
-1. Contributing Guidelines -- `.github/CONTRIBUTING.md`
-1. Issue Template -- `.github/ISSUE_TEMPLATE.md`
+1. Readme — `README.md`
+1. License — `LICENSE`
+1. Contributing Guidelines — `.github/CONTRIBUTING.md`
+1. Issue Template — `.github/ISSUE_TEMPLATE.md`
 
-## Readme
+## Readme File
```

Comment:
Unless we call it s license file below I'd rather not make this change

Reply:
also we don't say file for any of them above. my preference is to leave it.

Reply:
I think it'd be fine to rename section "license" to "license file" too. 

Or we could skip it both changes.

Reply:
Okay, so how about we revert the two changes and merge this in?